src/shrutiC/NumericOperations.java

```diff
@@ -0,0 +1,81 @@
+package shrutiC;
+
+public class NumericOperations {
+
+	void displayArray(int[] numberArray) {
+		System.out.print("Array : ");
+		for (int index = 0; index < numberArray.length; index++)
+			System.out.print(numberArray[index] + " ");
+	}
+
+	void displayReverseArray(int[] numberArray) {
+		System.out.print("\nReverse Array : ");
+		for (int index = numberArray.length - 1; index >= 0; index--)
+			System.out.print(numberArray[index] + " ");
+
+	}
+
+	int findMaximumNum(int[] numberArray) {
+
+		int maxNum = numberArray[0];
+		for (int index = 0; index < numberArray.length; index++) {
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files